python/ray/scripts/scripts.py

```diff
@@ -1926,6 +1927,93 @@ def cpp(show_library_path, generate_bazel_project_template_to, log_style,
                 " && bash run.sh"))
 
 
+@cli.group(hidden=True)
+@click.option(
+    "--address",
+    type=str,
+    default=None,
+    required=False,
+)
+@click.pass_context
+def job(ctx, address: Optional[str]):
```

Comment:
Should the format of this address be "head_node_ip:port" or "ray://head_node_ip:port"?

Reply:
let's do ray:// to match ray.init